src/models/fields/IllnessStartDate.schema.ts

```diff
@@ -0,0 +1,50 @@
+import * as Joi from '@hapi/joi';
+
+const illnessStartDayErrorMessage: string = 'You must enter a day';
+const illnessStartMonthErrorMessage: string = 'You must enter a month';
+const illnessStartYearErrorMessage: string = 'You must enter a year';
+const invalidDate: string = 'Enter a real date';
+const invalidDateFuture: string = 'Start date must be today or in the past';
+
+const dayMonthRegex: RegExp = /^[0-9]{1,2}$/i;
+const yearRegex: RegExp = /^[0-9]{2,4}$/i;
+
+export const schema = Joi.object({
+    startDay: Joi.string()
+        .required()
+        .pattern(dayMonthRegex)
+        .messages({
+            'any.required': illnessStartDayErrorMessage,
+            'string.base': illnessStartDayErrorMessage,
+            'string.empty': illnessStartDayErrorMessage,
+            'string.pattern.base': illnessStartDayErrorMessage
```

Comment:
doesn't your pattern cover the `string.base`, `string.empty` and `any.required`?

Reply:
i'll check this

Reply:
tried removing the above and the tests then fail, as an alternative error message is returned